__tests__/app.test.js

```diff
@@ -24,30 +25,61 @@ describe('GET /api/topics', () => {
 });
 
 describe('GET /api', () => {
-    test('STATUS 200: reponds with an object describing all the available endpoints on the API', () => {
+    test('STATUS 200: responds with an object describing all the available endpoints on the API', () => {
         return request(app)
         .get("/api")
         .expect(200)
         .then(result => {
+            const endpointsFileData = require("../endpoints.json")
             const {endpoints} = result.body
             const parsedEndpoints = JSON.parse(endpoints)
             expect(parsedEndpoints.length >= 1)
-            for (const endpoint in parsedEndpoints){
-                if (endpoint !== "GET /api") {
-                    // console.log(parsedEndpoints[endpoint])
-                    expect(typeof parsedEndpoints[endpoint].description).toBe("string")
-                    expect(Array.isArray(parsedEndpoints[endpoint].queries)).toBe(true)
-                    expect(typeof parsedEndpoints[endpoint].exampleResponse).toBe("object")
-                } else {
-                    expect(expect(typeof parsedEndpoints[endpoint].description).toBe("string"))
-                }
-            }
+            expect(endpointsFileData).toEqual(parsedEndpoints)
+            
         })
     });
 });
 
+describe('GET /api/articles:article_id', () => {
+    test('STATUS 200: responds with an article object, which should have its appropriate properties ', () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(result => {
+            const {article} = result.body
```

Comment:
The article should not be returned in an array back to the user. Arrays are for more than one item or potentially more than one item. We know that there will only every be one article returned so we should just return that. Please achieve this by rewriting your test then updating your code to pass it. TDD remember!